### PR TITLE
KYLIN-5289 Upgrade all modules for open-source

### PR DESCRIPTION
**Pull request by yhcast0**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Branch to commit
- [ ] Branch **kylin3** for v2.x to v3.x
- [ ] Branch **kylin4** for v4.x
- [x] Branch **kylin5** for v5.x

## Types of changes

What types of changes does your code introduce to Kylin?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on [Kylin's jira](https://issues.apache.org/jira/browse/KYLIN), and have described the bug/feature there in detail
- [x] Commit messages in my PR start with the related jira ID, like "KYLIN-0000 Make Kylin project open-source"
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at user@kylin.apache.org or dev@kylin.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yhcast0** [comment]
retest this please

**Comment by yhcast0** [comment]
retest this please

File: src/common-booter/src/main/resources/config/init.properties [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+## A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+## OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+## SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+## LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+## DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+## THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+## (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+## OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+##
 kylin.metadata.url=${KYLIN_METADATA_URL}
```
Comment (dethrive) [inline src/common-booter/src/main/resources/config/init.properties:24]: 不用改是吧？

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please.

**Comment by yhcast0** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please.

**Comment by Mukvin** [comment]
retest this please.

**Comment by Mukvin** [comment]
retest this please

**Comment by Mukvin** [comment]
retest this please

**Comment by yhcast0** [comment]
retest this please

**Comment by yhcast0** [comment]
image
CI Testing pass. Sonar failed due to script problems.

**Pull request merged**